src/pug/includes/body.pug

```diff
@@ -1,7 +1,9 @@
 include ./fork.pug
 
 .container
-
+	img(src="/assets/img/moon.png")#mode
```

Comment:
We could set this image using CSS background-image instead so this could be switched to single static div.

also, mode can be changed to something like `color-switch`